Swabbr.Api/Controllers/UserController.cs

```diff
@@ -57,6 +57,25 @@ public async Task<IActionResult> GetAsync([FromRoute] Guid userId)
             return Ok(output);
         }
 
+
+        // GET: api/user/follow-requesting-users
+        /// <summary>
+        ///     Get wrappers around all users which have a follow 
+        ///     request pending for our current user.
+        /// </summary>
+        [HttpGet("follow-requesting-users")]
+        public async Task<IActionResult> GetFollowRequestingUsersAsync([FromQuery] PaginationDto pagination)
+        {
+            // Act.
+            var userWithRelationWrappers = await _userService.GetFollowRequestingUsersAsync(pagination.ToNavigation()).ToListAsync();
```

Comment:
```suggestion
            ? userWithRelationWrappers = await _userService.GetFollowRequestingUsersAsync(pagination.ToNavigation()).ToListAsync();
```

Try avoid using `var`s, add explicit typing instead